### PR TITLE
Merge `master` into `stable`

### PR DESCRIPTION
**Pull request by nicholastmosher**

### CODE REVIEW

File: content/cli/commands/completion.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
```
Comment (sehz) [inline content/cli/commands/completion.md:1]: This is from https://github.com/infinyon/fluvio-website/commit/356c92144d84ab4ce6b96be06f28f83b135989b9 which should be omitted.

**Review by sehz — CHANGES_REQUESTED** [review]
looks like this h

File: content/cli/commands/completion.md (hunk 1 above)
Comment (nicholastmosher) [reply content/cli/commands/completion.md:1]: Why should that be omitted?

Comment (sehz) [reply content/cli/commands/completion.md:1]: it's not in current release

**Pull request merged**